server/controllers/admin/grades.js

```diff
@@ -56,16 +56,17 @@ function detail(req, res, next) {
 // POST /grade
 function create(req, res, next) {
   const data = req.body;
+  const recordUuid = data.uuid || uuid();
 
   // Provide UUID if the client has not specified
```

Comment:
The comment is now about convert into binary, for example : Convert record string uuid into a binary uuid